src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -55,7 +55,7 @@ Our correlation logic is called decisions. Learn how to set up decisions, follow
 
 ### Notify the right people at the right time [#workflow-intro]
 
-Once you've used our anomaly detection to identify irregular behavior and decisions to group all your relevant issues together, it's important that the right people on your team are notified of any potential problems. New Relic created workflows so you can control when and where you want to receive notifications about issues. Workflows allow you to funnel the right information to the relevant person or team. Workflows also give you the opportunity to enrich your notifications with additional New Relic data.
+Once you've used our anomaly detection to identify irregular behavior and correlation logic to group all your relevant issues together, it's important to notify the right people on your team of any potential problems. New Relic created workflows so you can control when and where you want to receive notifications about issues which allow you to funnel the right information to the relevant person or team. Workflows also give you the opportunity to enrich your notifications with additional New Relic data.
 
 To learn how to set up workflows, follow our step by step guide by [in our workflows docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
 
```

Comment:
I think there's a voice inconsistency on line 63 between "have been deprecated" and "are replaced". Maybe can just say "Pathways have been replaced by workflows"